openedx/features/content_type_gating/partitions.py

```diff
@@ -82,7 +82,8 @@ def access_denied_fragment(self, block, user, user_group, allowed_groups):
         course_key = self._get_course_key_from_course_block(block)
         modes = CourseMode.modes_for_course_dict(course_key)
         verified_mode = modes.get(CourseMode.VERIFIED)
-        if verified_mode is None or not self._is_audit_enrollment(user, block):
+        if (verified_mode is None or not self._is_audit_enrollment(user, course_key) or
+                user_group.name == 'Full-access Users'):
```

Comment:
Can we use the constants defined here https://github.com/edx/edx-platform/blob/master/openedx/features/content_type_gating/helpers.py#L15-L16?

Reply:
ah good idea will update